devops/docker/tasking-manager/Dockerfile

```diff
@@ -1,31 +1,44 @@
-# Base on Python on Debian stable
 FROM python:3-stretch
+MAINTAINER Tasking Manager development team
 
-EXPOSE 5000
+## API/SERVER CONFIGURATION
 
-# Install dependencies
+# Install dependencies for shapely
 RUN apt-get update \
   && apt-get upgrade -y \
   && apt-get install -y libgeos-dev \
   && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-RUN curl -sL https://deb.nodesource.com/setup_9.x | bash - \ 
+RUN mkdir -p /usr/src/app
```

Comment:
For better readability, I suggest to move this line and the next on top of the `## SERVER CONFIGURATION` block.

Reply:
Check